### ISSUE TITLE
Validation score

### ISSUE THREAD

**Issue by bloodymeli** [MVIG-SJTU/AlphaPose#210]
Hi,

I have loaded the pretrained model duc_se.pth in train.py and check the validation test score using:
loss, acc = valid(val_loader, m, criterion, optimizer, writer)

the resulting accuracy is very poor (0.54). Is this accuracy the same as the reported mAP? If not, how do I check the mAP of the pretrained model on COCO (and MPII)?

**Comment by Fang-Haoshu** [comment]
you need to use the coco api
